### PR TITLE
Re-enable commented code chunks in several notebooks

### PR DESCRIPTION
**Pull request by rmshaffer**
*Issue #, if available:*

*Description of changes:*
This is a resubmission of PR #372 (which was reverted in PR #414) after merging and resolving conflicts with #408.

The merge conflict resolution was limited to the file `2_Graph_optimization_with_QAOA.ipynb`. All other files are unchanged from PR #372.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by ajberdy — APPROVED** [review]

**Comment by krneta** [comment]
Sorry, can we hold off on merging this until after we're re-enabled the tests?

**Comment by krneta** [comment]
Hey Ryan, sorry about the long wait, but I think we've enabled the tests now where this should be working. Can you please verify?

**Comment by rmshaffer** [comment]
@krneta @ajberdy finished merging with latest changes from main and all tests are passing, please approve and merge if you agree that this is ready to go! Thanks for the help.

**Review by krneta — APPROVED** [review]

**Pull request merged**
